test-globalprotect-login.py

```diff
@@ -72,6 +78,7 @@
     xml = None
 
 if xml.tag == 'jnlp':
+    cert_and_key = ""
```

Comment:
Derp… thanks. :man_facepalming:

Reply:
@dlenski No problem - I think I've made the change you requested, but let me know if you meant something different.